03.bowling/bowling_score.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+score = ARGV[0]
+scores = score.split(',')
+
+shots = []
+scores.each do |s|
+  if shots.length < 18
+    if s == 'X'
+      shots << 10
+      shots << 0
+    else
+      shots << s.to_i
+    end
+  else
+    shots << if s == 'X'
+               10
+             else
+               s.to_i
+             end
```

Comment:
若干トリッキーですが、こうすれば三項演算子でも書けますね。ご参考まで。
（このケースでは、右辺の括弧がないとうまく動かないようです）

```suggestion
    shots << (s == 'X' ? 10 : s.to_i)
```